NOTICE.txt

```diff
@@ -1006,6 +1006,45 @@ Files covered:
 spyder/plugins/variableexplorer/widgets/dataframeeditor.py
 
 
+-------------------------------------------------------------------------------
+
+
+
+Classes from objbrowser 1.2.1
+-----------------------------
+
+
+Copyright (c) 2016 Pepijn Kenter
+
+
+Author: <author-name> | <author-email> | <author-website>
+Site/Source: https://github.com/titusjan/objbrowser
+License: MIT (Expat) License | https://opensource.org/licenses/MIT
+
+Modifications made to each class to adapt them to Spyder's needs.
+
+
+objbrowser is distributed under the MIT license.
+
+
+Use of the core functionality of the project changing the non-patched
+version of qtpy used for the actual qtpy project (since it is already a
+dependency of Spyder)
+
+See below for the full text of the MIT (Expat) license.
+
+The current gtabview license can be viewed at:
+https://github.com/titusjan/objbrowser/blob/master/LICENSE.txt
+
+The current version of the original files can be viewed at:
+https://github.com/titusjan/objbrowser/tree/master/objbrowser
+
+
+Files covered:
+
+spyder/plugins/variableexplorer/widgets/objecteditor2/
```

Comment:
We should be more specific and list the files covered, since the tests are not (and we don't want to misrepresent their authorship), nor the __init__.py files, nor anything we might add in the future.